metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/ELASTICSEARCH/2.3.3/configuration/elastic-site.xml

```diff
@@ -27,6 +27,14 @@
         <description>Cluster name identifies your cluster</description>
     </property>
     <property>
+        <name>masters_also_are_datanodes</name>
+        <value>"false"</value>
```

Comment:
No, it's not that important.  Can you add to the description that it has to be in quotes for ES compatibility?  I can definitely see that causing confusion otherwise.